python/pyspark/ml/clustering.py

```diff
@@ -56,8 +56,83 @@ def gaussiansDF(self):
         """
         return self._call_java("gaussiansDF")
 
+    @property
+    @since("2.0.0")
```

Comment:
Since we've shipped 2.0 we will need to update this to 2.1 (same with the versionAdded notes)